### PR TITLE
Add Kemai application

### PR DESCRIPTION
**Pull request by AlexandrePTJ**
Hi,
This is a proposal to integrate Kemai desktop client.

### CODE REVIEW

**Comment by kevinpapst** [comment]
Awesome, thanks @AlexandrePTJ  👍 

I saw there were some "work in progress for Mac" commits. Any chance that I can test this? 
Just clone and build? Or is it not yet ready? 

Any chance you could add some more information for the users.  E.g. which platforms are supported? Screenshots are great as well!
I don't mind if you keep all the other docs in your README, but some basic infos in the marketplace would be great, as most of the users are not devs. Lets make it a bit easier for them to understand what Kemai is and what it can and cannot do.

**Comment by AlexandrePTJ** [comment]
Thanks you for the feedback.
You can test it on mac. You "just" need Qt, cmake and xcode. I am working on a way to make it directly available on mac too.
Also I will do some screenshot to complete the description.

File: _store/alexandreptj-kemai-desktop-client.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+title: Desktop client for Kimai
+name: Kemai
+intro: Cross platform GUI client for Kimai
+developer: alexandreptj
+date: "2020-05-05 21:00:00 +0100"
+icon: fas fa-desktop
+price: "0"
+screenshot: 
+  - /images/marketplace/kemai-running.jpg
+  - /images/marketplace/kemai-settings.png
```
Comment (kevinpapst) [inline _store/alexandreptj-kemai-desktop-client.md:11]: .jpg

**Comment by kevinpapst** [comment]
Thanks for the screenshots, I almost thought you forgot about it 😁 

Thats how it will look like.
Bildschirmfoto 2020-05-05 um 23 42 07

I am working on another image slider, living in the content area of the page. I will update the page once it is ready.

File: _store/alexandreptj-kemai-desktop-client.md (hunk 1 above)
Comment (AlexandrePTJ) [reply _store/alexandreptj-kemai-desktop-client.md]: Fixed.

**Comment by AlexandrePTJ** [comment]
> 
> 
> Thanks for the screenshots, I almost thought you forgot about it 😁
> 
> Thats how it will look like.
> Bildschirmfoto 2020-05-05 um 23 42 07
> 
> I am working on another image slider, living in the content area of the page. I will update the page once it is ready.

Thanks you. I did not forget, just have to find some time to finish a first version ;)

**Comment by kevinpapst** [comment]
Thank you for the contribution @AlexandrePTJ 👍

**Comment by AlexandrePTJ** [comment]
Thank you for the merge.

**Pull request merged**